api/overdrive.py

```diff
@@ -746,50 +749,23 @@ def json(self):
         return json.loads(self.content)
 
 
-class DummyOverdriveAPI(OverdriveAPI):
+class DummyOverdriveAPI(MockOverdriveAPI, OverdriveAPI):
```

Comment:
That makes sense, how about making the naming match the two OverdriveAPI classes, so it's more clear that they're doing the same thing? Import MockOverdriveAPI as BaseMockOverdriveAPI and name both classes MockOverdriveAPI.